library/core/src/ptr/mut_ptr.rs

```diff
@@ -2078,6 +2128,7 @@ impl<T, const N: usize> *mut [T; N] {
 impl<T: ?Sized> PartialEq for *mut T {
     #[inline(always)]
     #[allow(ambiguous_wide_pointer_comparisons)]
+    #[cfg(not(feature = "ferrocene_certified"))]
```

Comment:
Can you please avoid marking methods as uncertified if the whole impl block is uncertified? 👉🏾👈🏾 

I am quite concerted with keeping the diff as small as possible.